lib/r10k/cli/deploy.rb

```diff
@@ -31,6 +31,13 @@ def self.command
             exit 1
           end
         end
+        option nil, :'puppet-conf', 'Path to puppet.conf', argument: :required do |value, cmd|
+          unless File.readable? value
```

Comment:
We have validation also defined on the setting itself, so this isn't strictly necessary. If we drop this validation, we get the following:

```
r10k deploy -c r10k.yaml --puppet-conf not_a_file environment
ERROR    -> Validation failed for 'global' settings group:
    deploy:
        Validation failed for 'deploy' settings group:
            puppet_conf:
                The specified puppet.conf not_a_file is not readable
```

Is the above error sufficiently clear or are we better served by this more specific error messaging?

Reply:
I'm not sure how to make the error more specific. The check is if the file is readable and the error states it's not readable. Not much else to check or tell the user about.

Reply:
@treydock I think what @adrienthebo is saying is that we can drop this `readable?` check altogether because there is validation elsewhere that will produce a similar error message.

Reply:
I removed the validation from this particular location.